client/src/cmdlfhitaghts.c

```diff
@@ -309,12 +326,93 @@ static int CmdLFHitagSRead(const char *Cmd) {
         if (page_addr > 255) {
             break;
         }
-        if (card->pages_reason[i] >= 0)
-            PrintAndLogEx(SUCCESS, "%02u | %s", page_addr, sprint_hex_ascii(card->pages[i], HITAGS_PAGE_SIZE));
-        else
+        if (card->pages_reason[i] >= 0) {
+            PrintAndLogEx(SUCCESS, "% 3u | %s  | " NOLF, page_addr, sprint_hex_ascii(card->pages[i], HITAGS_PAGE_SIZE));
+
+            // access right
+            if (page_addr == HITAGS_UID_PADR) {
+                PrintAndLogEx(NORMAL, _RED_("RO  ")NOLF);
```

Comment:
_RED_  for  `read only`  might be a bit too harsh.    How about we just let it be normal white?

Reply:
I'm try to following the lf hitag read --ht2 color scheme. Read-only is usually normal for the UID page. However, for other pages, it means they are locked.

Reply:
We need a better unified color scheme.